Update HTTP API doc (#2319)
infiniflow/infinity: docs/references/http_api_reference.mdx

---

### Admin remove node

**POST** `/admin/node/{node_name}`

> Only leader node can run this command, it will remove a node from the cluster.

Reviewer: ```suggestion
Removes a node from the cluster. This command can only be executed by the leader node.
```
